conconartist/lyric-lava
# Feature/form

#### Purpose: 
> Check all that apply.
### What does this PR do? 
> Clearly and concisely describe changes made. 

Added functionality to form components for searching rhyming words and searching similar words (synonyms).  Refactored minimize number of components and added layout changes.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

App.js, FormResults.js, RhymeForm.js, SynonymForm.js and Prompt.js were added and updated, along with corresponding CSS

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Clone this repo and run `npm install`.

Run `npm start` in project directory in terminal to see the layout and functionality.  

Click prompt button for random word to check functionality.  A new word/phrase should display.

Enter a word into search bars to see corresponding lists display.

### Steps To Take
> Any questions and possible steps moving forward.

Move API calls to apiCall file and perform E2E testing.

#### Checklist:
